discover/portal/articles/210-setting-up/01-system-settings/01-system-settings.markdown

```diff
@@ -0,0 +1,121 @@
+# System Settings [](id=system-settings)
+
+@product@ is modular, meaning it's composed of many applications, which are
+divided into even smaller "chunks" of functionality. The applications, and
+sometimes even code chunks, are configurable at several scopes, as discussed in
+the introductory article for this section. 
+
+To make configuration changes to an application in a system-wide fashion, use
+the System Settings application at Control Panel &rarr; Configuration &rarr;
+System Settings.
+
+![Figure 1: System Settings are accessed through the Control Panel.](../../../images/system-settings-categories.png)
+
+## Editing the Default Configuration [](id=editing-the-default-configuration)
```

Comment:
I'm a bit confused by this title. Why is it referring to the *default* configuration?

What is being changed here is the configuration in general. Now, when setting up the configuration at the virtual instance, site or widget scopes, the specific instances may change the configuration.

Reply:
https://dev.liferay.com/discover/portal/-/knowledge_base/7-0/system-settings#editing-the-default-configuration
It's a holdover from our 7.0 docs. I think we emphasize that System Settings sets the *default* configuration because setting a configuration at a different scope breaks the connection with System Settings. Once you configure a Site Setting called *my-setting*, no amount of System Settings changes to *my-setting* woll affect the site. You've overridden the *default* from System Settings. Is that incorrect, or should we de-emphasize that it's the default?

Reply:
Even if my understanding is correct, I do think the title of this section should be changed.

Reply:
Hey Russ,
What you are saying is correct, but we also need to take into account the system scope, for which the values are final. So saying that System Settings is used just to set default configuration is wrong IMO.